shared/test/test_sources.rb

```diff
@@ -95,7 +95,7 @@ def test_source_versions
 
   def test_sources_public
     filename = MAIN_JSON
-    file_data = '{"someData":"abc 123"}'
+    file_data = '{"source":"abc 123"}'
```

Comment:
It seems like we might be updating these values to move closer toward reflecting the actual data we see - is this correct?

Reply:
Yup! that's it exactly. I just noticed when I was updating the the unit tests for the files_api that the objects didn't have the `source` property that we expected